_data/simple-icons.json

```
@@ -12607,6 +12607,16 @@
             "hex": "EC008C",
             "source": "https://www.veepee.fr"
         },
+        {
+            "title": "Vega",
+            "hex": "2450B2",
+            "source": "https://github.com/vega/logos/blob/af32bc29f0c09c8de826aaafb037935fb69e960a/assets/VG_Black.svg",
+            "guidelines": "https://github.com/vega/logos",
+            "license": {
+                "type": "BSD-3-Clause",
+                "url": "https://github.com/vega/logos/blob/af32bc29f0c09c8de826aaafb037935fb69e960a/LICENSE"
```

```suggestion
                "type": "BSD-3-Clause",
```

Optional. This `url` line can be removed. We have some logic for generating the license link automatically.

https://github.com/simple-icons/simple-icons/blob/856435d461d12261456afb6530ee3a84113b3fcc/scripts/build/package.js#L52-L54

---

Good to know! Let's remove it then.